plugins/rbac-backend/src/service/policies-rest-api.test.ts

```diff
@@ -1152,6 +1164,21 @@ describe('REST policies api', () => {
       });
     });
 
+    it('should not create a role - name is invalid', async () => {
+      const result = await request(app)
+        .post('/roles')
+        .send({
+          memberReferences: ['user:default/permission_admin'],
+          name: 'x:default/rbac_admin',
+        });
+
+      expect(result.statusCode).toBe(400);
+      expect(result.body.error).toEqual({
+        name: 'InputError',
+        message: `Invalid role definition. Cause: Unsupported kind x. List supported values ["user", "group", "role"]`,
```

Comment:
I think message should be: 
```suggestion
        message: `Invalid role definition. Cause: Unsupported kind x. List supported values ["user", "group", "role"]`,
```
```suggestion
        message: `Invalid role definition. Cause: Unsupported kind x. Supported value should be "role"`,
```

Reply:
Ops. I didn't wanted to make commit...